src/libs/meta/meta.c

```diff
@@ -989,6 +1002,14 @@ static int topCmpOrder (const void * a, const void * b)
 	return strcmp (keyString (kam), keyString (kbm));
 }
 
+/*
+ * does topological sorting on "ks" using the dependencies defined by the "dep/#" metakeys
```

Comment:
That the depencencies are defined by the name? Ideally, you should write about how to use it and the specifics someone should know.